generated/cached-endpoints.edn

```diff
@@ -3522,7 +3522,7 @@
   :category ["Payment Services" "P2pLink API"]}
  {:path "paylink/{paylinkId}",
   :validOutputFormats ["json" "jsonp"],
-  :parameterDescriptions {:paylinkId "ID of the paylink to retrieve."},
+  :parameterDescriptions {:paylinkId "ID or UUID of the paylink to retrieve. UUID will be introduced on first week of September."},
```

Comment:
Sadly, we don't know the name of the version, so we can't say 'Available since SPiD version 2.111.0'. But 'on first week of September' is a bit confusing. I'd at least add 'September 2017' in case we forgot about it. And we could make this even a precise date. A precise date is always considered more professional than 'some time soon' :D
Also, I'd create a task in Jira to remember about removing this info as soon as it's there and replacing it with information about SPiD version since which it's available.
Anyways, the decision is up to Ricardo :)